settings.gradle

```diff
@@ -4,6 +4,7 @@ include ':financial-connections'
 include ':financial-connections-example'
 include ':identity'
 include ':identity-example'
+include ':hcaptcha'
```

Comment:
Have you tested this with a release build, with progaurd/r8 enabled? Do the code paths introduced still work?

Reply:
Not yet, I'll do that now.